pubsublite-beam-io/pom.xml

```diff
@@ -77,6 +77,11 @@
       <groupId>com.google.api.grpc</groupId>
       <artifactId>proto-google-cloud-pubsub-v1</artifactId>
     </dependency>
+    <dependency>
```

Comment:
This is for logging in the unbounded reader here:

https://codecov.io/gh/googleapis/java-pubsublite/pull/228/diff?src=pr&el=tree#D1-304

I thought it would be okay, because beam depends on this for it's gcp io modules here: https://github.com/apache/beam/blob/master/sdks/java/io/google-cloud-platform/build.gradle#L85

Is there something else I should be doing instead?

Reply:
Discussed a bit offline, changed to a test dependency